python/sglang/srt/managers/tp_worker_overlap_thread.py

```diff
@@ -110,7 +110,11 @@ def forward_thread_func_(self):
 
             # Run forward
             compute_info_done.wait()
-            logits_output, next_token_ids = self.worker.forward_batch_generation(
+            (
+                logits_output,
+                next_token_ids,
+                next_token_embeds,
```

Comment:
I think `next_token_embeds` is probably not necessary here. It makes things much more complicated.
Some of your handling here is not correct as you need to handle the copy of them correctly.
Ideally, we can get rid of `next_token_embeds` and do not need to change this file.